src/style.css

```diff
@@ -0,0 +1,131 @@
+.page {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.page__content {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__images {
+  margin: 32px 19px 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 16px 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__text {
+  margin: 0 16px 16px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+}
+
+.rewiew {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  text-align: end;
+  padding: 0 16px;
+  height: 16px;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  width: 96px;
+  height: 16px;
+}
+
+.stars_star {
+  display: inline-block;
+  background-image: url(~/images/star.svg);
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars_star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars_star:nth-child(-n+4) {
```

Comment:
better to add all cases (e.g. -n+3...) to keep it operational in case you change the block modifier